android/beagle/src/test/java/br/com/zup/beagle/android/components/WebViewTest.kt

```diff
@@ -78,6 +71,15 @@ class WebViewTest : BaseComponentTest() {
         assertFalse((stateSlot.captured as ServerDrivenState.Loading).loading)
     }
 
+    @Test
+    fun webViewClient_should_notify_when_page_WebViewError() {
+        val stateSlot = slot<ServerDrivenState>()
+        val webViewClient = createMockedWebViewClient(stateSlot)
+        webViewClient.onReceivedError(webView, null, null)
+        (stateSlot.captured as ServerDrivenState.WebViewError).retry.invoke()
+        verify(exactly = once()) { webView.reload() }
```

Comment:
Please separate into given, when and then to improve readability
```suggestion
       // Given
        val stateSlot = slot<ServerDrivenState>()
        val webViewClient = createMockedWebViewClient(stateSlot)
        
        // When
        webViewClient.onReceivedError(webView, null, null)
        (stateSlot.captured as ServerDrivenState.WebViewError).retry.invoke()
        
        // Then
        verify(exactly = once()) { webView.reload() }
```

Reply:
Change done!